xbmc/NfoFile.cpp

```diff
@@ -131,10 +131,10 @@ int CNfoFile::Load(const std::string& strFile)
 {
   Close();
   XFILE::CFile file;
-  XFILE::auto_buffer buf;
+  std::vector<char> buf;
```

Comment:
Hi, just a question from an interested user: Why not use `std::string` directly as the buffer? :)

At least in this file, it seems that actual characters are stored and not binary data (It's about NFO files after all).

Thanks.

Reply:
I've updated to use `std::string` I'm not sure if I like it or not yet as it seems a bit awkward in the usage of `CFile::LoadFile`.

Reply:
I don't mind any of both ways. I was just asking myself what the benefit of one vs the other was. :)

Reply:
Using `std::string` in the context of `CFile::LoadFile` seems very wrong to me. When loading arbitrary files who says there aren't `'\0'` in the middle and who guarantees there is a `'\0'` at the end?